app/models/passenger.rb

```diff
@@ -9,6 +9,10 @@ class Passenger < ApplicationRecord
   scope :active, -> { where(active: true) }
   scope :inactive, -> { where(active: false) }
 
+  # TODO: Make configurable by user
```

Comment:
Sounds good! We're going to tackle this in a different branch so there isn't too much going on in one branch at one time.